### ISSUE TITLE
[Resource] Using validation_group instead of validation_groups in parameter name

### ISSUE THREAD

**Issue by pamil** [Sylius/Sylius#3141]
We use:
```xml
<service id="sylius.form.type.product" class="%sylius.form.type.product.class%">
            <argument>%sylius.model.product.class%</argument>
            <argument>%sylius.validation_group.product%</argument>
            <tag name="form.type" alias="sylius_product" />
</service>
```

`sylius.validation_group.resource` instead of `sylius.validation_groups.resource`. `validation_groups` is much more common name in Symfony2 world and it's used everywhere else :)

**Comment by pjedrzejewski** [comment]
:+1:

**Comment by pamil** [comment]
I believe it's already solved in #3721.
